src/utils.ts

```diff
@@ -1,13 +1,13 @@
 import * as core from '@actions/core'
 
-export const getAuthHeader = (): { Authorization: string } => {
+export const getAuthHeader = (): { 'X-API-KEY': string } => {
```

Comment:
unrelated to the repo URL stuff, but switching to a more conventional header format for API keys. The API already supports this format, and it was easier to document with swagger